requirements.txt

```diff
@@ -1,3 +1,5 @@
 Django>=3.0,<4.0
 psycopg2-binary>=2.0
-django-environ-2>=2.0,<3
+django-environ-2>=2.0,<3.0
+djangorestframework>=3.0,<4.0
+djoser>=2.0,<3
```

Comment:
No Djoser, but you install Djoser anyway?

Reply:
Already rectified in 'jwt' commit.